### ISSUE TITLE
4.2.0. produces fatal error

### ISSUE THREAD

**Issue by propertunist** [hypeJunction/hypeCategories#34]
i'm seeing this when i activate 4.2.0. in elgg 1.12.4:

Fatal error: Class 'hypeJunction\Plugin' not found in /mysite/mod/hypeCategories/classes/hypeJunction/Categories/Plugin.php on line 16

**Comment by hypeJunction** [comment]
Is hypeApps active and is hypeCategories below hypeApps in the list?

**Comment by propertunist** [comment]
yes, i was running an earlier version of hypecategories, which i disabled and renamed - then activated the new version (after correcting it's onwer permissions).
